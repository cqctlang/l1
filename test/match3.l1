tab = [1:2,[3]:4];
switch (tab) {
  case 1: printf("error1!\n"); break;
  case [1:2]: printf("error2!\n"); break;
  @match [1:2,[4]:v]: printf("error3!\n"); break;
  @match [1:3,[3]:v]: printf("error4!\n"); break;
  @match [1:v,[3]:v2]: printf("got it (%d,%d)\n",v,v2); break;
}
