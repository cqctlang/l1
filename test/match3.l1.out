got it (2,4)
